ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -416,6 +416,27 @@ monitor server radius
    probe threshold failure 100
    probe method access-request username arista password 7 141600021F102B
 !
+ip name-server group mynameserver0
+   name-server vrf default 1.1.1.1
+   name-server vrf default 8.8.8.8
+   name-server vrf default 2.2.2.2 priority 1
+   dns domain arista.avd.com
+   ip domain-list domain-list1
```

Comment:
Can we give different values for `dns domain` and `ip domain-list` across list.

Reply:
done.